Projects/Contracts/Sources/Stores/TravelInsuranceStore.swift

```diff
@@ -23,7 +23,7 @@ final class TravelInsuranceStore: StateStore<TravelInsuranceState, TravelInsuran
                 }
                 let input = OctopusGraphQL.TravelCertificateCreateInput(
                     contractId: config.contractId,
-                    startDate: travelInsuranceModel.startDate.localDateString,
+                    startDate: travelInsuranceModel.startDate.displayDateDotFormat ?? "",
```

Comment:
This shouldn't be dot since it is input for mutation

Reply:
fixed